MediaAdminBundle/Form/Type/OrchestraSiteForFolderChoiceType.php

```diff
@@ -75,7 +75,7 @@ protected function getChoices()
         $userGroups = $this->tokenStorage->getToken()->getUser()->getGroups();
         /** @var GroupInterface $group */
         foreach ($userGroups as $group) {
-            if (in_array($group->getSite()->getSiteId(), $siteIds) && ($group->hasRole('ROLE_ACCESS_TREE_FOLDER'))) {
+            if (in_array($group->getSite()->getSiteId(), $siteIds) && ($group->hasRole('ROLE_ACCESS_MEDIA_FOLDER'))) {
```

Comment:
replace hasRole by method isGranted of authorizationChecker and
a group does not necessarily a site